assets/scss/local.scss

```diff
@@ -2,6 +2,22 @@
   min-height: 300px;
 }
 
+.env--dev .govuk-service-navigation__service-name::after {
```

Comment:
Adding env name to header just with CSS as there isn't really a way to get the actual content into the GOVUK page header component.